std/math.d

```diff
@@ -155,6 +155,18 @@ else version(D_InlineAsm_X86_64)
     version = InlineAsm_X86_Any;
 }
 
+version (X86_64) version = StaticallyHaveSSE;
+version (X86) version (OSX) version = StaticallyHaveSSE;
```

Comment:
I'm not sure if this is correct. Does OSX require SSE? This is here, because I've noticed that dmd emits SSE code for 32-bit OSX.